processor/src/main/java/com/linecorp/decaton/processor/ProcessorProperties.java

```diff
@@ -109,6 +109,39 @@
                     CONFIG_COMMIT_INTERVAL_MS,
                     CONFIG_GROUP_REBALANCE_TIMEOUT_MS));
 
+    /**
+     * Find and return a {@link PropertyDefinition} from its name.
+     * This method is not optimized for frequent invocation.
+     *
+     * @param name the name of property.
+     * @return a {@link PropertyDefinition} instance.
+     */
+    public static PropertyDefinition<?> definitionForName(String name) {
```

Comment:
Currently this and `propertyForName` are only used for `benchmark` module.
Do you imagine non-decaton-internal use case of these methods?

Reply:
ya, I think one of the most typical case that causing inconvenience handling is properties load from file. Users typically define their own class for mapping props into typed values and then converting it into `Property`s. 
I think it's good of decaton lib can support such work without needing them to define a copy of possible parameters to take but at least the added two methods are not enough (so I'm doing manual parsing in benchmark module). It will be too large to handle together within this PR but do you think adding those 2 methods here now causes any problem later on?

Reply:
Agree with this.
> I think it's good of decaton lib can support such work without needing them to define a copy of possible parameters to take

No, with follow-up PR :)
> do you think adding those 2 methods here now causes any problem later on?